assets/core/less/navbar.less

```diff
@@ -1,437 +1,439 @@
 /* Navbar
 -------------------------------------------------- */
-@dropdown-link-active-color:     #fff;
-@dropdown-link-active-bg:        @component-active-bg;
+@navbar-height:                    57px;
 
-@dropdown-link-hover-color:      #fff;
-@dropdown-link-hover-bg:         @dropdown-link-active-bg;
+#mainnav {
+  position: fixed;
+  width: 100%;
+  background: #004b80;
+  height: @navbar-height;
+  z-index: 10;
+}
+#mainnav .container { position: relative; }
+#mainnav .glyphicon { top: 0; }
 
-@navbar-height:                    57px;
+#mainnav .dropdown-menu {
+  margin: 0 0 0 -1px;
+  border-top: none;
+}
+#mainnav .dropdown-menu .divider { padding: 0; }
 
-@navbar-default-color:             #FFF;
-@navbar-default-bg:                #004b80;
+#mainnav .container > ul {
+  list-style: none;
+  display: inline-block;
+  vertical-align: middle;
+  margin: 0;
+  padding: 0;
+}
 
-@navbar-default-link-color:                #FFF;
-@navbar-default-link-hover-color:          #FFF;
-@navbar-default-link-hover-bg:             rgba(255,255,255,0.1);
-@navbar-default-link-active-color:         #FFF;
-@navbar-default-link-active-bg:            transparent;
+.mn-nav {
+  -webkit-transform: translate3d(0, 0, 0);
+}
 
-@navbar-default-brand-hover-color:         darken(@navbar-default-link-color, 10%);
-@nav-pills-active-link-hover-color:         #fff;
+.mn-nav li {
+  line-height: @navbar-height;
+  display: inline-block;
+  border-top: 5px solid rgba(255, 255, 255, 0.1);
+  margin: 0;
+  margin-right: -3px; /* removes space between inline-block elements */
+}
 
-@state-warning-text:             #c09853;
-@state-warning-bg:               #fcf8e3;
-@state-warning-border:           darken(spin(@state-warning-bg, -10), 3%);
+.mn-nav li > .online-logo-link {
+  padding: 0 13px 0 0;
+}
 
-@state-danger-text:              #b94a48;
-@state-danger-bg:                #f2dede;
-@state-danger-border:            darken(spin(@state-danger-bg, -10), 3%);
```

Comment:
I don't think these variables are actually unused. 
They overwrite Bootstrap variables and are probably not used by us directly. 
Odd place to overwrite this, but that's another issue. 

Do you know if stylelint takes into account less files imported from node_modules too?

Reply:
We had a quick chat on Slack, but for the sake of clarity:
I'm fairly sure the variables are unused, at least I can't spot a difference to the site when removing them. You're right on them overwriting Bootstrap, which is why I didn't spot their usage. Stylelint doesn't check for unused variables, that was just my own doing, which was partly why I put it in its own commit in case I was wrong. 
I'm somewhat :tm: sure that we can just remove them, but since I want to do a bigger sweep of unused styling in the future, I just moved them into bootstrap-variables.less.